src/main/kotlin/com/depromeet/makers/domain/usecase/UpdateMemberPassCord.kt

```diff
@@ -0,0 +1,18 @@
+package com.depromeet.makers.domain.usecase
+
+import com.depromeet.makers.domain.gateway.MemberGateway
+import com.depromeet.makers.domain.model.Member
+
+class UpdateMemberPassCord(
+    private val memberGateway: MemberGateway,
+) : UseCase<UpdateMemberPassCord.UpdateMemberPassCordInput, Member> {
+    data class UpdateMemberPassCordInput(
+        val memberId: String,
+        val passCord: String,
+    )
+
+    override fun execute(input: UpdateMemberPassCordInput): Member {
+        val member = memberGateway.getById(input.memberId)
+        return memberGateway.save(member.copy(passCord = input.passCord))
```

Comment:
```suggestion
        return memberGateway.save(member.copy(passCord = input.passCord))
```

도메인에 `initializePassCord(input.passCord)` 만들어서 넣어주는게 좋지 않을까여!!?!

Reply:
좋습니다!!